### ISSUE TITLE
Different number of points 

### ISSUE THREAD

**Issue by Amir-Abi** [fxia22/pointnet.pytorch#88]
Hi, point net was originally trained on a fixed-sized input , and I wonder if there is a efficient way to deal with different numbers of points in a batch. for example, (3, 10) and (3, 11) are my samples, how I should use them in the same batch ??? is there any better way than padding & passing mask that it'd used in NLP ???

**Comment by jasmineytt** [comment]
I got same question, how you solve the problem?

**Comment by Med-Amine-saighi** [comment]
I think you should implement a collate function and pass it to the DataLoader so you can get heterogonous batches
